views/interlibrary_loan_requests.erb

```diff
@@ -31,7 +31,7 @@
           <%= interlibrary_loan_request.creation_date %>
         </td>
         <td>
-          <%= interlibrary_loan_request.expiration_date %>
+          <a href="https://ill.lib.umich.edu/illiad/illiad.dll?Action=21&Type=10&Value=<%=interlibrary_loan_request.illiad_id%>">Cancel Request</a>
```

Comment:
You could make the url a method in the request model, and a corresponding one in the loan model.